src/users/collection/getItemsInFolderForUser.test.js

```diff
@@ -22,8 +22,13 @@ describe('Users - Collection - getItemsInFolderForUser', () => {
     })
   })
   it('should get items in a folder in a user\'s collection from its username and a `folderId`', async () => {
-    const data = await client.getItemsInFolderForUser(username, folderId)
+
+    const data = await client.getItemsInFolderForUser({ username, folderId, pagination: { perPage: 1 } })
```

Comment:
This won't pass as you are passing an object to the function (see other PR).

Reply:
removed object, oops